Tests/SwiftMemcacheTests/IntegrationTest/MemcachedIntegrationTests.swift

```diff
@@ -84,4 +85,50 @@ final class MemcachedIntegrationTest: XCTestCase {
             XCTFail("Failed to connect to Memcached server: \(error)")
         }
     }
+
+    func testMemcachedConnectionActor() async throws {
+        let ELG = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+        let connectionActor = MemcachedConnection(host: "memcached", port: 11211, eventLoopGroup: ELG)
+        Task {
+            try await connectionActor.run()
+        }
+
+        let setValue = "foo"
+        var setBuffer = ByteBufferAllocator().buffer(capacity: setValue.count)
+        setBuffer.writeString(setValue)
+        let _ = try await connectionActor.set("bar", value: setValue)
+
+        // Get value for key
+        let getValue = try await connectionActor.get("bar")
+        let getValueString = getValue?.getString(at: getValue!.readerIndex, length: getValue!.readableBytes)
+        XCTAssertEqual(getValueString, setValue, "Received value should be the same as sent")
+        try await ELG.shutdownGracefully()
```

Comment:
Can you please always add this in a `defer` right after you create the ELG. This makes sure that we are not leaking the ELG. You can also use `try!` in there.

Reply:
Please use `syncShutdown()` in a `defer` instead

Reply:
this didn't even cross my mind, pushing this up now! thank you